lmdeploy/pytorch/configurations/builder.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from abc import ABC
+
+from lmdeploy.utils import get_logger
+
+logger = get_logger('lmdeploy')
+
+
+class AutoModelConfigBuilder(ABC):
+
+    _sub_classes = list()
```

Comment:
why not use dict in here for quickly querying?

Reply:
What key?

Reply:
string of class name?

Reply:
It is meaningless, we would only use the class when `condition` is true.